test/acceptance/features/customEnvVar.feature

```diff
@@ -0,0 +1,128 @@
+Feature: Inject custom env variable into application
+
+  As a user of Service Binding Operator
+  I want to inject into application context an env variable
+  whose value whose value might be generated from values available in service resources
+
+  Background:
+    Given Namespace [TEST_NAMESPACE] is used
+    * Service Binding Operator is running
+
+  Scenario: Sequence from service resource is injected into application using custom env variables without specifying annotations
+    Given OLM Operator "backend" is running
+    * The Custom Resource is present
+            """
+            apiVersion: "stable.example.com/v1"
+            kind: Backend
+            metadata:
+                name: backend-with-tag-sequence
+            spec:
+                host: example.common
+                tags:
+                    - "centos7-12.3"
+                    - 123
+            """
+    * Generic test application "foo" is running
+    When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: custom-env-var-from-sequence
+            spec:
+                application:
+                    name: foo
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: backend-with-tag-sequence
+                    id: backend
+                customEnvVar:
+                   - name: TAGS
+                     value: '{{ .backend.spec.tags }}'
+            """
+    Then The application env var "TAGS" has value "[centos7-12.3 123]"
```

Comment:
Do we need `[` for the value
Also should we check the status conditions to be True ?

Reply:
`[centos7-12.3 123]` is the value that gets injected.

Reply:
oh the value is in the form of a list